src/rgw/rgw_sal_motr.cc

```diff
@@ -366,6 +366,43 @@ int MotrUser::read_stats(const DoutPrefixProvider *dpp,
     ceph::real_time *last_stats_sync,
     ceph::real_time *last_stats_update)
 {
+  int rc, max_entries = 100; // to fetch in chunks of 100
+  vector<string> keys(max_entries);
+  vector<bufferlist> vals(max_entries);
+  std::string user_stats_iname = "motr.rgw.user.stats." + info.user_id.to_str();
+  rgw_bucket_dir_header bkt_header;
+
+  do
+  {
+      rc = store->next_query_by_name(user_stats_iname, keys, vals);
+      if (rc < 0) {
+          ldpp_dout(dpp, 20) << __func__ << ": failed to get the user stats info for user  = "
+                            << info.user_id.to_str() << dendl;
+          return rc;
+      }
+
+    for (int i = 0 ; i < rc; i++)
```

Comment:
rc is written code which is not meaningful variable name in case of successful operation at line 377. Please create a variable that reflects "number of entries" if rc is not less than 0 and use it throughout the code.

Reply:
Addressed